great-idea/css/index.css

```diff
@@ -62,3 +69,77 @@ h1, h2, h3, h4, h5 {
     letter-spacing: 1px;
     margin-bottom: 15px;
 }
+
+h1 {
+	font-size: 80px;
+}
+
+.container {
+	width: 880px;
```

Comment:
Adding a width to the container is a great way to make a predictable layout that will look the same every time. As we move into responsive layouts, I encourage you to think about the pros and cons of a fixed-width content container versus a responsive layout (or even just a different CSS property, like adding margins to the container).